packages/blade/src/components/TopNav/TabNav/TabNavItem.web.tsx

```diff
@@ -105,7 +106,7 @@ const _TabNavItem: React.ForwardRefRenderFunction<HTMLAnchorElement, TabNavItemP
     as,
     children,
     isActive,
-    leading: Leading,
+    icon: Icon,
     trailing: Trailing,
```

Comment:
nit: this doesn't need to TitleCased now. Got a little confused that its still a component

Reply:
done.